pkg/validators/test_001_default_channel.go

```diff
@@ -0,0 +1,28 @@
+package validators
+
+import (
+	"github.com/mt-sre/addon-metadata-operator/internal/testutils"
+	"github.com/mt-sre/addon-metadata-operator/pkg/utils"
+)
+
+// check interface implemented
+var _ = utils.ValidatorTest(ValidatorTest001DefaultChannel{})
+
+type ValidatorTest001DefaultChannel struct{}
+
+func (v ValidatorTest001DefaultChannel) Name() string {
+	return "Addon Default Channel Validator"
+}
+
+func (v ValidatorTest001DefaultChannel) Run(mb utils.MetaBundle) (bool, string, error) {
+	return Validate001DefaultChannel(mb)
+}
+
+func (v ValidatorTest001DefaultChannel) SucceedingCandidates() []utils.MetaBundle {
+	return testutils.DefaultSucceedingCandidates()
+}
+
+// not implemented
+func (v ValidatorTest001DefaultChannel) FailingCandidates() []utils.MetaBundle {
+	return []utils.MetaBundle{}
```

Comment:
shouldn't we add the previously added failingCandidates here as well for now ?

Reply:
done